charts/newrelic-prometheus-agent/values.yaml

```diff
@@ -182,8 +182,23 @@ config:
   # -- It allows defining scrape jobs for Kubernetes in a simple way.
   # @default -- See `values.yaml`
   kubernetes:
-    jobs:
-    # By default all pods and endpoints with `prometheus.io/scrape: true` annotation will be discovered.
+    # NR has a list of Dashboards, alerts and entities for several Services and the curated_experience configuration
+    # allows to keep only the targets having this experience out of the box.
+    # Notice that this configuration option applies to all jobs having a prefix matching the jobs_prefix list.
+    # If curated_experience is enabled, then the jobs, selected by jobs_prefix, scrapes merely the targets having one of the specified labels matching
+    # one of the values of app_values.
+    # Under the hood relabel_configs with action 'keep' are generated, consider it in case any custom extra_relabel_config is needed.
+    curated_experience:
+      # -- enabling the curated experience the jobs, selected by jobs_prefix, scrapes merely the targets having one of the specified labels matching
+      #    # one of the values of app_values.
+      enabled: true
+      # -- source_labels to be used in the relabel configs when looking for app_values.
+      source_labels: ["app.kubernetes.io/name", "app.newrelic.io/name"]
```

Comment:
I think `"app.newrelic.io/name"` was needed befor the new solution adding the extra job filtering by `newrelic.io/scrape: true` so i think we could remove it.

Also i think `k8s-app` is missing here perhaps (anyway we need to add the final list before merge

Reply:
I think it could be still useful to have a newrelic labels that can be used and that creates no conflicts in the cluster. (imagine that for any reason they cannot change the app label)

Moreover, the dashboard were changed to take that into account, therefore I would keep it, however I added a point to discuss it again before merging